### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by tasnimulhasan07**
create Html file for mobile skeleton using containers and background images and nav bar than using CSS to position and style each element, a file included all files required eg install linters and gitignore files, and added images for page

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+}
+
+.homepage {
+  background-image: url("image/backimg.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+  margin: auto;
+  overflow: hidden;
+  min-height: 600px;
+}
```
Comment (prolajumokeoni) [inline styles.css]: - [x] Kindly implement requested changes on the previous [pull request open
](https://github.com/tasnimulhasan07/scel2/pull/1)   the previous reviewer requested you make the background image look like the Figma design, check screenshot below ⬇️

Figma design           |  Your design
:-------------------------:|:-------------------------:
designB | bgimg
 

**_Please do not open a new Pull Request, You should use this same Pull Request for your  next submission when asking for a review_**

File: index.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+            <div class="connect">
+                <h3><a> Let's connect </a> </h3>
+            </div>
+            <br>
+            <div class="social">
+                <ul>
+                    <li class="icon"><a href=""><img src="Image/twitter.png" alt="" /></a></li>
+                    <li class="icon"><a href=""><img src="Image/github.png" alt="" /></a></li>
+                    <li class="icon"><a href=""><img src="Image/medium.png" alt="" /></a></li>
+                    <li class="icon"><a href=""><img src="Image/angellist.png" alt="" /></a></li>
```
Comment (prolajumokeoni) [inline index.html]: ## Optional

Figma Design          |  Your Design
:-------------------------:|:-------------------------:
image   |  smicons

- Kindly add the missing icon, so your design looks like the figma design, as this was missed by the previous reviewer 👍🏿

File: index.html [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+    </header>
+    <br>
+    <br>
+    <section class="homepage">
+        <div class="introduction">
+            <div class="intro"><h1>I'm Tasnimul <br>Glad to meet you!</h1></div>
+            <br>
+            <br>
+            <br>
```
Comment (prolajumokeoni) [inline index.html]: ### OPTIONAL

- [x] Kindly remove all br tags and use margin and padding(top, bottom, left, right) to make your design look like the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) this also helps you understand the [css box model](https://www.w3schools.com/css/css_boxmodel.asp) 👍🏿

File: index.html [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+                        <span class="bar"></span>
+                        <span class="bar"></span>
+                        <span class="bar"></span>
+                    </div>
+            </nav>
+        </div>
+
+    </header>
+    <br>
+    <br>
```
Comment (prolajumokeoni) [inline index.html]: ### OPTIONAL

- [x] Kindly remove all br tags and use margin and padding(top, bottom, left, right) to make your design look like the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) this also helps you understand the [css box model](https://www.w3schools.com/css/css_boxmodel.asp) 👍🏿

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @tasnimulhasan07 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Highlights
- Github flow was used correctly ✔️ 
- Good readme ✔️ 

 ## Required Changes ♻️
- [x] Some changes were required by the previous reviewer kindly check inline comments to see the ones you missed and implement them, do not open a new Pull Request for re-reviews. You should use this same Pull Request for the next review.
- [x] Kindly use a descriptive pull request title explaining changes inside pull request 👍🏿 

## Optional suggestions

**_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**



almost-there

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (tasnimulhasan07) [reply styles.css]: br has been removed and padding and margin has been added as per request

File: index.html (hunk 2 above)
Comment (tasnimulhasan07) [reply index.html]: linked in image has been added to page

File: index.html (hunk 3 above)
Comment (tasnimulhasan07) [reply index.html]: br has been removed

File: index.html (hunk 4 above)
Comment (tasnimulhasan07) [reply index.html]: br has been removed

**Review by CollinsTatang — APPROVED** [review]
Hi @tasnimulhasan07, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**